src/Sylius/Bundle/AdminBundle/Resources/config/routing/shipment.yml

```diff
@@ -12,3 +12,16 @@ sylius_admin_shipment:
             index:
                 icon: truck
     type: sylius.resource
+
+sylius_admin_shipment_ship:
```

Comment:
Can't we reuse `sylius_admin_order_shipment_ship` route?

Reply:
`sylius_admin_order_shipment_ship` is done with an `updateAction`, that means we can't use it with the `apply_transition` Grid action (as far as I know), which means it would not be, for example, hidden if no transition is possible